recordium/config.py

```diff
@@ -41,6 +41,7 @@ class _Config(object):
         'BOT_AUTH_TOKEN': '',
         'POLLING_TIME': 30,
         'USER_ALLOWED': None,
+        'COL_ORDER': '2, 0, 1, 3'
```

Comment:
Why not have a list with ints here? it will simplify the code elsewhere...

Also, default order should be what's current choice.